docs/FAQ/FAQ-General.md

```diff
@@ -46,129 +44,50 @@ BTCPay doesn't compromise on censorship-resistance, which is one of the main fea
-- Polis (POLIS)
-- Viacoin (VIA)
-
- Altcoins are maintained and supported by respective communities.
-
 ## Why can't I just give my Bitcoin address to a buyer?
 
-Re-using the address for receiving payments is a privacy issue. Providing a different address manually to each customer is not an optimal solution. Imagine having to send an e-mail to everyone that wants to pay you with cryptocurrency.
+Re-using an address for receiving payments is a privacy issue. Providing a different address manually to each customer is not an optimal solution. Imagine having to send a unique e-mail to everyone that wants to pay you with cryptocurrency.
 
-BTCPay solves the address re-use and automates the checkout process. The software is a complete, automated invoicing system. It integrates into your checkout process, and customers can pay you in Bitcoin or altcoins in a few clicks.
+BTCPay solves the address re-use issue. It automates the checkout process for the merchant by creating a new invoice with a unique address created from the merchant's wallet, each time a customer pays using BTCPay. If you are using an integration with an e-commerce store, BTCPay Server integrates into your checkout process, and customers can pay you in Bitcoin or altcoins in a few clicks, just like any other traditional payment option.
 
-After the payment, the software notifies your store that the order has been paid/completed. Depending on the e-commerce software you're using, it can also change the order status. All you have to worry about is getting items shipped, leave the invoicing and payment processing to BTCPay.
+After a customer makes a payment, the BTCPay Server software notifies your store that the order has been paid/completed. Depending on the e-commerce software you're using, it can also change the order status. All you have to worry about is getting items shipped, leave the invoicing and payment processing to BTCPay.
 
-## Does BTCPay need my private key?
+## How does BTCPay create a new address for each invoice?
 
-Absolutely not. The fact that BTCPay Server never needs access to your master private key for on-chain transactions is a huge security advantage. Even if your server gets hacked, your funds from the on-chain transactions are always safe. Securing your on-chain funds comes down to [securing your wallet](https://btcinformation.org/en/secure-your-wallet).
+Merchants connect a wallet to each of their stores that they want to receive payments to. The invoices produced in a store are linked directly to the merchant's connected wallet. The invoice addresses are derived from the [xpubkey](https://bitcointalk.org/index.php?topic=2828777.0) of the wallet associated with the store. The software only needs your extended public key of a wallet to produce new addresses.
 
-If you have a lightning node, BTCPay technically has access to the keys (macarons) of your LN.
+## Does BTCPay need my private key?
```

Comment:
Uh, we now have Hot Wallet option, so "absolutely not" is not the best term here. We can probably re-word this section entirely.

Some context https://docs.btcpayserver.org/WalletSetup/#use-an-existing-wallet  
https://docs.btcpayserver.org/HotWallet/#security-implications

Reply:
That one is tough because we don't want to confuse new users to think that it is "required".. let me know what you think of this update